src/test/resources/schemaOrgJsonTestFiles/schemaOrg_40221e30-1414-11e9-8fb8-00505688346e.json

```diff
@@ -1 +1 @@
\ No newline at end of file
+{"@context":["http:\/\/schema.org\/",{"kb":"http:\/\/kb.dk\/vocabs\/","relator":"https:\/\/id.loc.gov\/vocabulary\/relators\/"}],"@type":"Photograph","id":"40221e30-1414-11e9-8fb8-00505688346e","url":"https:\/\/example.com\/imageserver\/\/DAMJP2\/DAM\/Samlingsbilleder\/0000\/624\/420\/KE070592","kb:admin_data":{"kb:catalogingLanguage":"da","kb:localIdentifier":"KE070592.tif","kb:shelfLocator":"Billedsamlingen. Danske portrætter, 2°, Christian 8 (1786-1848), 11"},"headline":[{"value":"Christian VIII","@language":"da"}],"creator":[{"@type":"Person","name":" Ubekendt litograf","familyName":"Ubekendt litograf"}],"contentLocation":[{"@type":"Place","description":"Danmark"}],"about":[{"@type":"Person","name":" Christian 8","familyName":"Christian 8","birthDate":"1786-9-18","deathDate":"1848-1-20","hasOccupation":{"@type":"Occupation","name":"konge af danmark (1839-1848)"}},"F. 1885"],"keywords":["Danske portrætter","Danmarks regenter","Christian 08"],"dateCreated":"1848","temporal":"Created between 1820 and 1848","publication":{"@type":"PublicationEvent","description":["1820","1848"]},"identifier":[{"@type":"PropertyValue","PropertyID":"UUID","value":"40221e30-1414-11e9-8fb8-00505688346e"},{"@type":"PropertyValue","PropertyID":"ShelfLocator","value":"Billedsamlingen. Danske portrætter, 2°, Christian 8 (1786-1848), 11"},{"@type":"PropertyValue","PropertyID":"localIdentifier","value":"KE070592.tif"},{"@type":"PropertyValue","PropertyID":"accessionNumber","value":"1911-14241"}],"isPartOf":[{"@type":"Collection","headline":"Billedsamlingen","description":"Det Kgl. Biblioteks billedsamling","url":"https:\/\/www.kb.dk\/find-materiale\/samlinger\/billedsamlingen\/index.html"}],"image":{"@type":"ImageObject","contentSize":"152932120","contentURL":"https:\/\/example.com\/imageserver\/\/DAMJP2\/DAM\/Samlingsbilleder\/0000\/624\/420\/KE070592","height":"8141","width":"6261","dateCreated":"2018-12-12T13:36:42.000+01:00"},"kb:read_direction":"LTR"}
```

Comment:
The image URL has a double slash: https://example.com/imageserver//DAMJP2/DAM/Samlingsbilleder/0000/624/420/KE070592 . Technically it works, but it ain't pretty. I know it's my fault for putting the slash in the properties, but maybe remove it from the properties and add a comment saying it should not end in slash? Or the fancy version: Remove the slash in the XSLT if it is present?